Sage/carbon
# fix(button): add border radius to button styles - FE-3681

fixes: #3630

### Proposed behaviour
Button has correctly set border-radius.
Edge default border-radius does not apply to Button.
Buttons do not have rounded corners.

### Current behaviour
Edge sets default border-radius to button elements which causes our Button to appear with rounded corners.

### Checklist


- [ ]<del> Screenshots are included in the PR if useful
- [ ] <del>Cypress automation tests added or updated if required

### Additional context


### Testing instructions
Open https://carbon.sage.com/?path=/story/design-system-button--primary in edge and see if buttons have rounded corners.
